pages/common/yacas.md

```diff
@@ -0,0 +1,28 @@
+# yacas
+
+> Yet Another Computer Algebra System.
+> More information: <http://www.yacas.org>.
+
+- Start an interactive console session:
+
+`yacas`
+
+- In the console session, execute a statement:
+
+`In> {{Integrate(x)Cos(x);}}`
+
+- In the console session, display an example:
+
+`In> {{Example();}}`
+
+- Quit from the console session:
+
+`In> {{quit}}`
+
+- Execute one or more yacas scripts (without terminal or prompts), then exit:
+
+`yacas -p -c {{path/to/script1}} {{path/to/script2}}`
+
+- Execute and print the result of one statement, then exit:
```

Comment:
```suggestion
- Calculate and print the result of one statement, then exit:
```
```suggestion
- Compute and print the result of one statement, then exit:
```
I think one of those two would fit better.

Reply:
yacas is not a calculator. The author's description is that yacas "executes statements", which are very general. For example "Echo()". You would not describe calling the printf function as "calculate" or "compute".